packages/remark-shiki-twoslash/src/index.ts

```diff
@@ -35,6 +45,9 @@ function getHTML(code: string, lang: string, metaString: string, highlighters: H
       return renderCodeToHTML(code, lang, metaString.split(" "), { themeName }, highlighter, twoslash)
     })
     results = output.join("\n")
+    if (highlighters.length > 1 && wrapFragments) {
+      results = `<div class="shiki-twoslash-fragment">${results}</div>`
+    }
```

Comment:
I see, nice thinking

Reply:
yes, I switched that div with a React fragment when rendering back in React